Create a photo gallery page

AmilaIndika789 opened the pull request:
I have created a photo gallery page which could include albums of various events and sports held in the college. Added a side bar and quotes of several famous sportsmen.  At the bottom there is a button named find more, thought of using it to link to photographers page. Any suggestions or comments?

AmilaIndika789 committed:
Design a photo gallery page

This page displays photo albums of various events and sports in a particular college.

AmilaIndika789 committed:
Fix an issue with npm package

Fixed the issue with npm package @zeit/next-css

shyaman commented on package.json:
```diff
@@ -29,6 +29,7 @@
     "react-dom": "^16.6.3"
   },
   "devDependencies": {
+    "@zeit/next-css": "^1.0.1",
```
Is this a dev dependency?

AmilaIndika789 replied:
@shyaman No I tried to remove, I haven't used it actually, just added during testing to add a feature. When I tried to remove it, there was a travis build failure. Check if it is possible to remove it without build failure

AmilaIndika789 replied:
@shyaman Seems like issue is resolved, removed the unwanted @zeit/next-css package without any build failure. Please check and resolve if problem still exists.

shyaman commented on pages/photos.js:
```diff
@@ -0,0 +1,968 @@
+                <div className="fh5co-footer">
+                    <p>
+                        <small>&copy; 2018 PeraSports, UOP. All Rights Reserved.
+                            <span>Designed by <a href="http://freehtml5.co/" target="_blank">Engineering Faculty, UOP</a> </span>
```
let's add this link here [http://www.ce.pdn.ac.lk/](http://www.ce.pdn.ac.lk/)

AmilaIndika789 replied:
@shyaman Added the link, resolving the conversation

shyaman commented on pages/photos.js:
```diff
@@ -0,0 +1,968 @@
+                    <p>
+                        <small>&copy; 2018 PeraSports, UOP. All Rights Reserved.
+                            <span>Designed by <a href="http://freehtml5.co/" target="_blank">Engineering Faculty, UOP</a> </span>
+                            <span>Images: <a href="http://pexels.com/" target="_blank">Pera Beats</a></span>
```
We can add perabeats fb page

AmilaIndika789 replied:
@shyaman Added the link, resolving the conversation

AmilaIndika789 committed:
Fix issue with @zeit/next-css, add links

Fixed the issue with the npm package @zeit/next-css and added the urls for Department of Computer Engineering and Pera Beats Facebook page.
